build-tools/automation/azure-pipelines.yaml

```diff
@@ -87,6 +87,13 @@ stages:
       parameters:
         exclusionFile: $(System.DefaultWorkingDirectory)\build-tools\automation\PoliCheckExclusions.xml
 
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-antimalware.AntiMalware@3
```

Comment:
What is `@3`?  I've seen `@yaml` before to pull in files from xamarin/yaml-templates, but `@3` is new to me and this file, so I'm not at all sure where this comes from or where to find the source.

Reply:
`@3` is the major version of the task, which allows for changes to the task to be released without breaking existing usage. Every task that is using a full name (not a shorthand name like `script` or `powershell`) will have a trailing `@$version`. This is seen elsewhere in the pipeline:
https://github.com/xamarin/xamarin-android/blob/8f5d32969c70f3347af019fac57df28139cd3a23/build-tools/automation/azure-pipelines.yaml#L214-L236